packages/scandipwa/src/component/CheckoutGuestForm/CheckoutGuestForm.container.js

```diff
@@ -137,10 +137,13 @@ export class CheckoutGuestFormContainer extends PureComponent {
             emailValue,
             isEmailAvailable,
             isLoading,
-            signInState
+            signInState,
+            onSignIn: this.handleOnMyAccountSignIn
         });
     }
 
+    handleOnMyAccountSignIn() {}
+
```

Comment:
I assume you can just set default property in CheckoutGuestForm.component with `noopFn` function as default value.

Reply:
yeah, it is actually done in a container, so I just need to pass the 'onSingIn' prop